services/QuillLMS/client/app/bundles/Evidence/components/activitySlides/postActivitySlide.tsx

```diff
@@ -14,6 +14,45 @@ interface PostActivitySlideProps {
   responses: any
 }
 
+// Helper function that can be exported for testing
+export function getResponsesForConjunction(responses: any, conjunction: string) {
+  const responsesKeys = Object.keys(responses);
+
+  for (const key of responsesKeys) {
+    const responseArray = responses[key];
+    if (!responseArray || !responseArray.length) continue;
+
+    const lastResponse = responseArray[responseArray.length - 1];
+    if (!lastResponse || !lastResponse.entry) continue;
+
+    const entry = lastResponse.entry;
+    const targetConjunctionRegex = new RegExp(`\\s${conjunction}\\s`);
+
+    if (!targetConjunctionRegex.test(entry)) continue;
+
+    const conjunctionMatch = entry.match(targetConjunctionRegex);
+    if (!conjunctionMatch) continue;
+
+    const conjunctionPosition = conjunctionMatch.index;
+
+    let hasOtherConjunctionBefore = false;
+    for (const otherConjunction of conjunctions) {
+      if (otherConjunction === conjunction) continue;
+
+      const otherConjunctionRegex = new RegExp(`\\s${otherConjunction}\\s`);
+      const otherMatch = entry.match(otherConjunctionRegex);
+
+      if (otherMatch && otherMatch.index < conjunctionPosition) {
+        hasOtherConjunctionBefore = true;
+        break;
+      }
+    }
+
+    if (!hasOtherConjunctionBefore) { return responseArray; }
```

Comment:
These other conjunction regexes are also going to be the same in every iteration, so you could do something like:

```
const otherConjunctionRegexes = conjunctions.filter(conj => conj !== conjunction).map(otherConjunction => new RegExp(`\\s${otherConjunction}\\s`))
```

at the top of this function definition, and then inside your iterative loop after `conjunctionPosition` is defined, you can just do:

```
const hasOtherConjunctionBefore = otherConjunctionRegexes.find(otherConjunctionRegex => {
  const otherMatch = entry.match(otherConjunctionRegex)
  return otherMatch && otherMatch.index < conjunctionPosition
}
```